dags/openshift_nightlies/scripts/install/prebuilt.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+set -ex
+
+while getopts u:p:w: flag
+do
+    case "${flag}" in
+        u) user=${OPTARG};;
+        p) pass=${OPTARG};;
+        w) url=${OPTARG};;
+    esac
+done
+
+create_login_secrets(){
+    echo ${user}; echo ${pass}; echo ${url};     
+    curl -sS https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/openshift-client-linux.tar.gz | tar xz oc
+    export PATH=$PATH:$(pwd)
+    
+    mkdir workspace
+    cd workspace 
+    export KUBECONFIG=kubeconfig
+    oc login -u ${user} -p ${pass} ${url} --insecure-skip-tls-verify
+    
+    unset KUBECONFIG
```

Comment:
why are we exporting the kubeconfig not using it then unsetting it 2 lines later? Or am I missing something?